Update the readme to include updated import and Faire's announcement (#5)
Faire/mjml-react: README.md

## Annoucement from Engineering @ Faire
Hello mjml community, we at Faire are excited to announce we will be taking over maintaining the open-source mjml-react project!

> Over the past year, Faire has used mjml-react extensively. This package allowed us to incorporate all the benefits of mjml without having to ask our developers to learn a new dev environment moving from web to email. As a result, we significantly reduced the dev effort for creating emails, while keeping email template issues to a minimum.

Reviewer: ```suggestion
Over the past year, Faire has used mjml-react extensively. This package allowed us to incorporate all the benefits of mjml without having to ask our developers to learn a new dev environment when moving from web to email. As a result, we significantly reduced the development effort for creating emails, while keeping email rendering issues to a minimum.
```

> We saw so much benefit from miml-react, and we don’t want to see it go. That is why we want to continue to maintain the package, and even improve it with some of our own new ideas! For now you can check us out at LINK for a drop in replacement for miml-react. From there you can look forward to more improvements in the future.

Reviewer 2: I have an update for this paragraph in our channel. Also the link to mjml-react is https://mjml.slack.com/archives/C043G87R7T9

> We saw so much benefit from mjml-react, and we don’t want to see it go. That is why we want to continue to maintain the package, and improve it in with some of our own new ideas! For now you can check us out at [@faire/mjml-react](https://github.com/Faire/mjml-react). We hope this can be a drop-in replacement in most cases, but are eager to work with you if there are any issues. Don’t hesitate to reach out to us at [#mjml-react](https://join.slack.com/t/mjml/shared_invite/zt-gqmwfwmr-kPBnfuuB7wof5httaTcXxg) or issues. From there, you can look forward to new improvements coming soon.

Reviewer: ```suggestion
We saw so much benefit from mjml-react, and we don’t want to see it go. That is why we want to continue to maintain the package, and improve it with some of our own new ideas! For now you can check us out at [@faire/mjml-react](https://github.com/Faire/mjml-react). We hope this can be a drop-in replacement in most cases, but are eager to work with you if there are any issues. Don’t hesitate to reach out to us on the #mjml-react channel in the [MJML Slack Group](https://join.slack.com/t/mjml/shared_invite/zt-gqmwfwmr-kPBnfuuB7wof5httaTcXxg) or file a [github issue](https://github.com/Faire/mjml-react/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). From there, you can look forward to new improvements coming soon.
```

> And of course we want to extend a massive thank you to the Wix team for building and maintaining this project.

Reviewer: ```suggestion
And of course we want to extend a massive thank you to the Wix team for building and maintaining this project.

```
